README.md

```diff
@@ -298,6 +300,22 @@ If no `error` is supplied, the `response` value can be any plain JavaScript
 value sent by the server, including any values that can be unserialized from
 [JSON].
 
+<a name="session.listen" />
+
+---
+
+> *`void`* [**`session.listen`**](#session.listen) `(...namespaces)`
+
+Listens to notifications on the given `namespaces` variable argument list.
+
+<a name="session.unlisten" />
+
+---
+
+> *`void`* [**`session.unlisten`**](#session.unlisten) `(...namespaces)`
+
+Unlistens from notifications on the given `namespaces` variable argument list.
```

Comment:
Once again, we probably need to make note of the fact that you can call unlisten safely with namespaces that you're not already listening to.